sunpy/conftest.py

```diff
@@ -128,11 +128,11 @@ def sunpy_test_run(request):
 @pytest.fixture(scope='session', autouse=True)
 def hide_parfive_progress(request):
     """
-    Set the PARFIVE_HIDE_PROGESS to hide the parfive progress bar in tests.
+    Set the PARFIVE_HIDE_PROGRESS to hide the parfive progress bar in tests.
     """
-    os.environ["PARFIVE_HIDE_PROGESS"] = "True"
+    os.environ["PARFIVE_HIDE_PROGRESS"] = "True"
     yield
-    del os.environ["PARFIVE_HIDE_PROGESS"]
+    del os.environ["PARFIVE_HIDE_PROGRESS"]
```

Comment:
errrrrm

Reply:
We have misspelt this either here for ages or parfive has.

Reply:
I sent the following message on Element on 2022-Oct-20:
> @Cadair: Um... As far as I can tell, parfive checks the environment variable PARFIVE_HIDE_PROGRESS (https://github.com/Cadair/parfive/blob/c75353cee89715a6eb1bfd9dce704a13cbd0ba52/parfive/config.py#L59), but no external code appears to set the correct environment variable. Instead, they have the misspelled PARFIVE_HIDE_PROGESS (https://github.com/search?q=PARFIVE_HIDE_PROGESS&type=code, and this misspelling is in the parfive docs) or HIDE_PARFIVE_PROGESS (https://github.com/search?q=HIDE_PARFIVE_PROGESS&type=code). Amusingly, sunpy core uses both misspellings.